blog.html

```diff
@@ -22,6 +22,286 @@
+                        <br>
+                        With KIP-714, cluster operators can now collect client metrics directly from brokers using a plugin.
+                        This makes it much easier to monitor client behavior. However, KIP-714 only gathers metrics from
+                        Kafka clients (admin, consumer, and producer) and doesn't cover application-level metrics for
+                        embedded clients, such as Kafka Streams. To fully understand performance, it's important to collect
+                        both client metrics and application-specific metrics. This KIP introduces a way for applications that
+                        use Kafka clients to include their own metrics alongside the existing client metrics.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/NIyMEw">KIP-1106: Add duration based offset reset option for consumer clients</a>
+                        <br>
+                        Kafka consumer supports auto.offset.reset config option, which is used when there is no initial
+                        offset in Kafka or if the current offset does not exist any more on the server. This config currently
+                        supports earliest/latest/none options. Currently consumer resets might force applications to
+                        reprocess large amounts of data from earlier offsets. With infinite storage, its beneficial to have a
+                        duration based offset reset strategy. This KIP allows applications to consume/initialise from a fixed
+                        duration when there is no initial offset in Kafka.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/XoowEg">KIP-1043: Administration of groups</a>
+                        <br>
+                        KIP-848 and KIP-932 introduce new group types, consumer and share respectively. The Admin Client
+                        API has limitations when dealing with newer group types, sometimes returning errors or incorrectly
+                        stating that groups don't exist. To address this challenge a new command-line tool, kafka-groups.sh,
+                        and updates to existing command line tools, kafka-consumer-groups.sh and kafka-share-groups.sh,
+                        were made in Kafka 4.0 to enable users to view all groups in a cluster, along with their types and
+                        protocols, providing accurate information even when the Admin Client API fails.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/w4mMEw">KIP-1099: Extend kafka-consumer-groups command line tool to support new consumer group</a>
+                        <br>
+                        This KIP extends the kafka-consumer-groups and the kafka-share-groups command line tools to
```

Comment:
Not sure if we want to use the `.sh` form of the command line tools consistently, or no.

```suggestion
                        This KIP extends the <code>kafka-consumer-groups.sh</code> and the <code>kafka-share-groups.sh</code> command line tools to
```